### PR TITLE
Fix typos in docs, comments and variables

### PR DESCRIPTION
**Pull request by alexandear**

### CODE REVIEW

**Review by domenicrosati — DISMISSED** [review]
ship it

File: cmd/heighliner/zz_generated_data.go [hunk 1]
```diff
@@ -371,7 +371,7 @@ func RestoreAssets(dir, name string) error {
 }
 
 func _filePath(dir, name string) string {
-	cannonicalName := strings.Replace(name, "\\", "/", -1)
-	return filepath.Join(append([]string{dir}, strings.Split(cannonicalName, "/")...)...)
+	canonicalName := strings.Replace(name, "\\", "/", -1)
```
Comment (domenicrosati) [inline cmd/heighliner/zz_generated_data.go]: we shouldn't change this file because its generated

**Review by domenicrosati — CHANGES_REQUESTED** [review]
Thanks for submitting a PR! awesome. Only one small request about the generated file

**Comment by alexandear** [comment]
@domenicrosati fixed

**Review by jelmersnoeck — DISMISSED** [review]

**Review by domenicrosati — DISMISSED** [review]

**Comment by jelmersnoeck** [comment]
Hey @alexandear ! Thanks for this. Apologies for the delay.

We've fixed the Travis CI parts in master, would you be able to update your branch?

**Comment by alexandear** [comment]
Update branch.

**Review by jelmersnoeck — APPROVED** [review]

**Pull request merged**